src/main/groovy/com/palantir/python/miniconda/MinicondaPlugin.groovy

```diff
@@ -141,4 +143,13 @@ class MinicondaPlugin implements Plugin<Project> {
             }
         }
     }
+
+    private static List<String> convertChannelsToArgs(List<String> channels) {
+        List<String> args = new ArrayList<>();
```

Comment:
I'm planning to eventually move this to Java instead of Gradle, so I would prefer to keep it like this.